dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterClient.java

```diff
@@ -109,13 +109,15 @@
 
   private ConfigCenterConfigurationSourceImpl.UpdateHandler updateHandler;
 
-  private static ClientPoolManager<HttpClientWithContext> clientMgr;
+  private ClientPoolManager<HttpClientWithContext> clientMgr;
 
   private boolean isWatching = false;
 
-  private static final ServiceLoader<AuthHeaderProvider> authHeaderProviders =
+  private final ServiceLoader<AuthHeaderProvider> authHeaderProviders =
       ServiceLoader.load(AuthHeaderProvider.class);
 
+  private URIConst uriConst = new URIConst();
```

Comment:
Do we need to create a instance of Constant object here?

Reply:
not necessary
only use once here

Reply:
After change, it's more convenient for testing different scenarios. We can benefit more :-)